orangecontrib/timeseries/widgets/owtabletotimeseries.py

```diff
@@ -66,9 +66,10 @@ def set_data(self, data):
         if data.domain.has_continuous_attributes():
             vars = [var for var in data.domain.variables if isinstance(var, TimeVariable)] + \
                    [var for var in data.domain.variables
-                    if var.is_continuous and not isinstance(var, TimeVariable)]
+                    if var.is_continuous and not isinstance(var,
+                                                            TimeVariable)] + \
+                   [var for var in data.domain.metas if var.is_time or var.is_continuous]
```

Comment:
`TimeVariable` is a `ContinuousVariable`, so `var.is_continuous` check should be enough.